### DISCUSSION TITLE
New Discord Link

### DISCUSSION THREAD

**Discussion by aidanleung0** [bym-refitted/backyard-monsters-refitted#96] [General]
Hello, I hope the devs find this! Sorry for using this as a way of communicating, but the discord link in the website is expired. The README is also outdated - the React page directory seems to have been reorganized. Could anyone with Discord access refresh the invite link? Thanks!

**Comment by React1-X** [comment]
> Hello, I hope the devs find this! Sorry for using this as a way of communicating, but the discord link in the website is expired. The README is also outdated - the React page directory seems to have been reorganized. Could anyone with Discord access refresh the invite link? Thanks!

Hey @aidanleung0 

Here is the permanent discord link:
https://discord.gg/bymrefitted
